### ISSUE TITLE
Incorrect/Out of date String [Spanish]

### ISSUE THREAD

**Issue by fiazk** [NewEconomyMovement/NEMiOSApp#248]
Not sure if this was an issue in previous versions but it is now. The button string displays "Mensaje de la" instead of "Mensaje de Factura" (Invoice Message) see images below. I had a look to the iOS loc. file and I could not find this string so it might be a very old one.
iOS: See "Mensaje de la"
![photo_2016-06-15_09-32-06](https://cloud.githubusercontent.com/assets/17099341/16072989/09232a46-32e4-11e6-9616-e3643ab023ea.jpg)

Android: See "Mensaje de factura"
![photo_2016-06-15_10-34-54](https://cloud.githubusercontent.com/assets/17099341/16073174/f7694104-32e4-11e6-8808-17e83c4fe449.jpg)

NEM Address: NDAZM2-KCD2XB-4Y7XMM-J5UUZ3-2ZCBHE-MHRHF4-3U3W

**Comment by fiazk** [comment]
@fyahfox @jabo38 it seems the iOS loc file was not updated into this version I have found a couple more strings that are out of date

**Comment by jabo38** [comment]
Yes, we haven't updated.  If you need to know for sure which line needs to be updated, change the langauge in the app to "Debug" and then you know the string title.  Also @fiazk can you highlight the strings you updated.
